enterprise/api/v1/views.py

```diff
@@ -213,6 +213,79 @@ def course_enrollments(self, request, pk):
 
         return Response(serializer.errors, status=HTTP_400_BAD_REQUEST)
 
+    @action(detail=True, methods=['post'], permission_classes=[permissions.IsAuthenticated])
+    @permission_required('enterprise.can_enroll_learners', fn=lambda request, pk: pk)
+    # pylint: disable=invalid-name,unused-argument
+    def enroll_learners_in_courses(self, request, pk):
+        """
+        Creates a set of licensed enterprise_learners by bulk enrolling them in the specified course.
+        """
+        enterprise_customer = self.get_object()
+        serializer = serializers.EnterpriseCustomerBulkSubscriptionEnrollmentsSerializer(
+            data=request.data,
+            context={
+                'enterprise_customer': enterprise_customer,
+                'request_user': request.user,
+            }
+        )
+        if serializer.is_valid(raise_exception=True):
+            errors = []
+            enrolled_count = 0
+            license_uuids = serializer.validated_data.get('emails')
```

Comment:
perhaps it's just the naming but we get a field called `emails` which is assigned to `license_uuids`. not sure but is it best to rename the emails field to sth like `license_info` or something? what you have further down is clear . e.g, `course_run_keys = serializer.validated_data.get('courses')`

also I think even if you keep this as is, perhaps having a quick example json in the docstring will help make it easier to see this?